### PR TITLE
[DNM]server: batch allocate timestamp

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
You should update Godeps.json.

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @shenli

File: pd-client/rpc_worker.go [hunk 1]
```diff
@@ -203,13 +203,16 @@ func (w *rpcWorker) handleRequests(requests []interface{}, conn *bufio.ReadWrite
 			ok = false
 			log.Error(err)
 		}
-		for i, req := range tsoRequests {
+		logicalHigh := ts.GetLogical()
+		physical := ts.GetPhysical()
+		for _, req := range tsoRequests {
 			if err != nil {
```
Comment (shenli) [inline pd-client/rpc_worker.go]: Why check err here?

**Comment by shenli** [comment]
LGTM

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**